Core/Src/driver_settings/dynamic_controls_choice.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * choose_dynamic_controls.c
+ *
+ *  Created on: 7 de jul de 2022
+ *      Author: JoaoPedroNunes
+ */
+
+#include "driver_settings/dynamic_controls_choice.h"
+
+#include "cmsis_os.h"
+#include "driver_settings/RTD.h"
+#include "util/CMSIS_extra/global_variables_handler.h"
+#include "util/buttons_handler.h"
+#include "util/global_instances.h"
+#include "util/global_variables.h"
+#include "util/util.h"
```

Comment:
```suggestion
#include "cmsis_os.h"
#include "util/global_instances.h"
#include "util/global_variables.h"
#include "util/util.h"
```

Sempre tenha apenas os includes necessários. Bom para segurança do código pq limita o escopo a acessar apenas o necessário. Reduz tempo de compilação tbm (eu acho)